### DISCUSSION TITLE
navigating internal provider networks and national wrap networks

### DISCUSSION THREAD

**Discussion by DHSDesigner** [CMSgov/price-transparency-guide#503] [Q&A]
Can someone comment on how to present output/presentation regarding in-network files for provider networks and national wrap networks (which may be considered in-network). Example: Let's say we have 6 provider networks and a few national networks. Could we create files for each (1 set of files for reach provider network and 1 for each of the national networks, then utilize the TOC file to identify priority? Do those internal networks need to be named and if so, how? Could we use priority pseudonyms for public consumption or would we be expected to identify those networks using some other method?

**Comment by DHSDesigner** [comment]
@shaselton-usds, can  you comment on this request?

**Comment by shaselton-usds** [comment]
@DHSDesigner 

You can create a file per network if you wish. You would referencing the external network within the in-network file, not the table of contents. The referenced networks will require an ID to be assigned to them by you (or the builder of the file) and then referenced later on in the in-network file.

You can find more information on how to use them in this discussion: https://github.com/CMSgov/price-transparency-guide/discussions/369#discussioncomment-2099481

**Reply by RLTx1391** [reply]
Wouldn't this be more appropriate to use the use case of external in-network files: i.e. like the schema example of a shared behavioral in-network file. It would be simpler for implementation to include the in-network file generated by a wrap network vendor and then reference it in the TOC file.
